json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -352,12 +356,18 @@ public function update_settings() {
 					}
 					break;
 				case 'wga':
+				case 'jetpack_wga':
 					if ( ! isset( $value['code'] ) || ! preg_match( '/^$|^UA-[\d-]+$/i', $value['code'] ) ) {
 						return new WP_Error( 'invalid_code', 'Invalid UA ID' );
 					}
-					$wga = get_option( 'wga', array() );
+
+					$is_wpcom = ! $is_jetpack && defined( 'IS_WPCOM' ) && IS_WPCOM;
```

Comment:
This triggers a PHP Notice when saving the tracking ID on wpcom:

`Undefined variable: is_jetpack in /wp-content/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php on line 364`